### ISSUE TITLE
Default values for config

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#615]
I need to have some default option values in `resolve.config.json` to use them in dev mode. Now the only way is to specify these variables *always*. Make something like this:
```
{
  "port": "${PORT}",
  "defaults": {
    "port": 4000
  }
}
```

**Comment by TatyanaRyzh** [comment]
Your suggestion of default values for config sounds interesting. Thank you for sharing it with us. We are always open to our users' feedback.

I'll share your proposal with the team. In case we implement it in a future release, we will let you know here.

Thank you for helping us improve reSolve!

**Comment by TatyanaRyzh** [comment]
Implemented in [0.14.0](https://github.com/reimagined/resolve/releases/tag/V0.14.0)
